sdk/src/client/node_api/indexer/routes.rs

```diff
@@ -84,7 +84,7 @@ impl Client {
             .get_output_ids(&route, QueryParameters::empty(), true, false)
             .await?
             .first()
-            .ok_or_else(|| crate::client::Error::Node("no output id for foundry".to_string()))?))
+            .ok_or_else(|| Error::NoOutput("foundry", foundry_id.to_string()))?))
```

Comment:
@kwek20 I meant why not like this?
```suggestion
            .ok_or_else(|| Error::NoOutput(format!("{foundry_id:?}")))?))
```

Reply:
Wouldnt you still want to know what type of output? this would just give the id.

Reply:
Because of :? it prints FoundryId(0x..) no?

Reply:
oh true! Smart

Reply:
Hm that would leave us unable to do filter on just foundry though. Or i need to split up the error but that seems excessive
```
match err  {
     Ok() => {}
     Err(crate::client::Error::NoOutput("foundry")) => {}
}
```

Reply:
Where do we need that? `Err(crate::client::Error::NoOutput(_))` is not fine?